### PR TITLE
NxLoadError retry styling - RSC-137

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-137

### CODE REVIEW

**Comment by rpokorny** [comment]
Note that the alignment of the icon is a little bit off in IE11 due to lack of support for `align-items: first baseline`.  I'm willing to live with that

File: lib/src/components/NxLoadError/NxLoadError.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.nx-load-error__message {
+  // TODO no magic numbers
+  max-width: calc(100% - 32px);
```
Comment (glenhunter) [inline lib/src/components/NxLoadError/NxLoadError.scss:3]: I noticed that in cases where the button wraps down (very clever solution BTW), but the text is long enough to read the right edge of the box it doesn't look very good.
Screen Shot 2020-06-11 at 2 32 19 PM
However if you add 8px of margin-bottom to `.nx-load-error__message` it adds some breathing space but doesn't effect non-wrapping instances.
Screen Shot 2020-06-11 at 2 46 07 PM

Comment (glenhunter) [inline lib/src/components/NxLoadError/NxLoadError.scss:3]:
```suggestion
  max-width: calc(100% - 32px);
```
  margin-bottom: $nx-spacing-sm;
  max-width: calc(100% - 32px);

Comment (rpokorny) [reply lib/src/components/NxLoadError/NxLoadError.scss]: > However if you add 8px of margin-bottom to .nx-load-error__message it adds some breathing space but doesn't effect non-wrapping instances.

631fb79

**Pull request merged**